src/components/uielements/CasePreview.astro

```diff
@@ -1,133 +1,111 @@
+  ]}>
     <div class="flex flex-col gap-2">
       <div class="mb-4 flex flex-row items-start justify-between gap-8">
         <div class="flex flex-col">
-          <span class="font-serif text-primary" set:html={caseData.client} />
+          <span class="font-serif text-primary" set:html={caseData?.metadata?.title} />
           <h2
             class="caseheadline mb-2 font-serif text-xl [&>mark]:bg-none [&>mark]:bg-clip-border [&>mark]:text-black"
-            set:html={caseData.title}
+            set:html={caseData?.intro?.title}
           />
         </div>
         {
-          caseData.logo && (
-            <Image
+          caseData?.metadata?.image && (
+            <SanityImg
               loading="lazy"
-              src={logoimages[caseData.logo]()}
+              src={caseData?.metadata?.image?.asset?.url as SanityImageSource}
               class="h-12 w-auto shrink-0 sm:h-16 md:h-12 xl:h-16"
-              loading="lazy"
               height={60}
-              alt={caseData.client || caseData.title}
+              alt={caseData?.metadata?.image?.asset?.altTexts?.[language] || ""}
             />
           )
         }
       </div>
-      <p
-        class="text-sm font-light [&>mark]:text-black"
-        set:html={caseData.description}
-      />
+      {      
+        caseData.intro?.intro && (
+          <div class="text-sm font-light [&>mark]:text-black">
+            <PortableText value={caseData.intro?.intro} />
+          </div>
+        )
+      }
       <ol class="mt-2 list-inside list-disc font-serif text-sm">
         {
-          caseData.duration && (
+          caseData.duration != null && (
             <li class="">
               {t("metadata.duration") + ": " + caseData.duration}
             </li>
           )
         }
         {
-          caseData.collaboration && (
+          caseData.collaborationModel && (
             <li class="">
-              {t("metadata.collaboration") + ": " + caseData.collaboration}
+              {t("metadata.collaboration") + ": " + caseData.collaborationModel.collaborationTabs.map(tab => tab?.concept?.prefLabel).join(', ')}
             </li>
           )
         }
         {
           caseData.technologies && (
             <li class="">
-              {t("metadata.technologies") + ": " + caseData.technologies}
+              {t("metadata.technologies") + ": " + caseData.technologies.map(technology => technology.name).join(', ')}
             </li>
           )
         }
         {
           caseData.services && (
             <li class="">
-              {t("metadata.services") + ": " + caseData.services}
+              {t("metadata.services") + ": " + caseData.services.map(service => service.metadata?.title).join(', ')}
             </li>
           )
         }
       </ol>
     </div>
     <Button
       class="self-end"
-      link={link}
-      style="primary"
+      link={`/preview/en/cases/${caseData?.metadata?.slug?.current}`}
```

Comment:
use the isPreview variable here and also the siteLanguage variable like in other files to make it dynamic.